Reqnroll/Infrastructure/DefaultDependencyProvider.cs

```diff
@@ -113,6 +113,7 @@ public virtual void RegisterTestThreadContainerDefaults(ObjectContainer testThre
             // needs to invoke methods so requires the context manager
             testThreadContainer.RegisterTypeAs<StepArgumentTypeConverter, IStepArgumentTypeConverter>();
             testThreadContainer.RegisterTypeAs<StepDefinitionMatchService, IStepDefinitionMatchService>();
+            testThreadContainer.RegisterTypeAs<MatchArgumentCalculator, IMatchArgumentCalculator>();
```

Comment:
Could you please move the registration to the `RegisterGlobalContainerDefaults` method? I think there is no need to have this bound to the test-thread (execution worker), but it can be global singleton.